lib/wallet/src/Cardano/Wallet/Write/Tx/Balance.hs

```diff
@@ -221,6 +220,8 @@ import qualified Cardano.Wallet.Primitive.Types.UTxO as UTxO
 import qualified Cardano.Wallet.Primitive.Types.UTxOIndex as UTxOIndex
 import qualified Cardano.Wallet.Primitive.Types.UTxOSelection as UTxOSelection
 import qualified Cardano.Wallet.Shelley.Compatibility.Ledger as W
+import qualified Cardano.Wallet.Write.Tx as Write
```

Comment:
Please import `Write` non-qualified in this module (`Write.Balance`) instead

Reply:
:ok-hand: